package.json

```diff
@@ -86,9 +90,17 @@
     "webpack-hot-middleware": "^2.12.2"
   },
   "jest": {
-    "testRegex": "(/test/.*\\.spec.js)$",
+    "testRegex": "(/test/unit/.*\\.spec.js)$",
     "setupFiles": [
-      "test/env.js"
+      "test/unit/env.js"
+    ],
+    "collectCoverageFrom": [
```

Comment:
The js files which need do unit test are distributed. The same situation as enslint checking.